docker-run.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Instead of requiring users to build the docker image we should have a `Dockerfile.build` file and just publish the image on quay.io. That would have the merit to:
- avoid building the image to build the image :-)
- `docker run ...` would be the build command, and that's is easier to understand and customise (for example replacing `docker` with `buildah`)

Reply:
But that can be done in another PR

Reply:
I agree 100 % with that, knowing that this image would also be usable in a `dockerImage` component of the repo `devfile.yaml` to allow testers to run the build and the test their changes themselves.

But I didn't want to wait for this before submitting this PR.